Source/Core/Common/MemArenaUnix.cpp

```diff
@@ -28,30 +28,39 @@ MemArena::~MemArena() = default;
 void MemArena::GrabSHMSegment(size_t size)
 {
   const std::string file_name = "/dolphin-emu." + std::to_string(getpid());
-  fd = shm_open(file_name.c_str(), O_RDWR | O_CREAT | O_EXCL, 0600);
-  if (fd == -1)
+  m_shm_fd = shm_open(file_name.c_str(), O_RDWR | O_CREAT | O_EXCL, 0600);
```

Comment:
If dolphin crashes, this file will stay until the system is rebooted. This is because it has the pid appended to it, so no other instance can come later and clean it up.

Reply:
Not wrong, but it's unlinked literally the next line, so it would have to crash at a *very* specific point in time. Not sure if there's much we can do about this? This is also a pre-existing issue.

Reply:
I completely missed that, then it's ok. Feel free to disregard this.